x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/data/AbstractBlock.java

```diff
@@ -23,6 +23,8 @@ abstract class AbstractBlock implements Block {
 
     protected final BlockFactory blockFactory;
 
+    protected boolean released; // false
```

Comment:
`= false` isn't slower and communicates the same thing as the comment I hink.

Reply:
Done